packages/react/src/components/Loading/Loading-test.js

```diff
@@ -6,31 +6,66 @@
  */
 
 import React from 'react';
-import Loading from '../Loading';
-import { mount } from 'enzyme';
-import { render, cleanup } from '@carbon/test-utils/react';
-import { afterEach } from 'jest-circus';
-
-const prefix = 'cds';
+import Loading from './Loading';
+import { render, screen } from '@testing-library/react';
 
 describe('Loading', () => {
-  afterEach(cleanup);
+  describe('renders as expected - Component API', () => {
+    it('should change classes based on active', () => {
+      const { container, rerender } = render(<Loading active />);
 
-  describe('automated accessibility testing', () => {
-    it('should have no Axe violations', async () => {
-      const { container } = render(<Loading />);
-      await expect(container).toHaveNoAxeViolations();
+      expect(container.firstChild).not.toHaveClass(
+        'cds--loading-overlay--stop'
+      );
+
+      rerender(<Loading active={false} />);
+
+      expect(container.firstChild).toHaveClass('cds--loading-overlay--stop');
     });
 
-    it('should have no AC violations', async () => {
-      const { container } = render(<Loading />);
-      await expect(container).toHaveNoACViolations('Loading');
+    it('should support a custom `className` prop on the outermost element', () => {
+      const { container } = render(
+        <Loading className="custom-class" withOverlay={false} />
+      );
+
+      expect(container.firstChild).toHaveClass('custom-class');
     });
+
+    it('should spread extra props on the outermost element', () => {
+      const { container } = render(
+        <Loading withOverlay={false} data-testid="test" />
+      );
+
+      expect(container.firstChild).toHaveAttribute('data-testid', 'test');
+    });
+
+    it('should specify a description based on prop', () => {
+      render(<Loading description="Loading description" />);
+
+      expect(screen.getByTitle('Loading description')).toBeInTheDocument();
+    });
+
+    it('should respect small prop', () => {
+      const { container } = render(<Loading small withOverlay={false} />);
+
+      expect(container.firstChild).toHaveClass('cds--loading--small');
+    });
+
+    it('should respect withOverlay prop', () => {
+      const { container, rerender } = render(<Loading withOverlay />);
+
+      expect(container.firstChild).toHaveClass('cds--loading-overlay');
+
+      rerender(<Loading withOverlay={false} />);
+
+      expect(container.firstChild).not.toHaveClass('cds--loading-overlay');
+    });
+  });
+  describe('behaves as expected', () => {
+    // Add tests for relevant component behavior. For more information, visit https://github.com/carbon-design-system/carbon/issues/10184#issuecomment-992978122
```

Comment:
Do we need this block if there are no relevant tests, or should we keep it for possible future enhancements?

Reply:
Nah, I just forgot to remove it. Thanks for reminding me!